src/main/java/me/playbosswar/com/CommandTimerPlugin.java

```diff
@@ -70,7 +72,8 @@ public void onEnable() {
         conditionEngineManager = new ConditionEngineManager();
         eventsManager = new EventsManager(tasksManager);
         inventoryManager.init();
-
+        GsonConverter gson = new GsonConverter();
```

Comment:
I don't think this gson converted is used here

Reply:
sorry i forgot to delete this